Misc/NEWS.d/next/IDLE/2018-03-03-12-56-26.bpo-32989.FVhmhH.rst

```diff
@@ -0,0 +1,2 @@
+Correct signature in call from editor to
+:meth:`pyparse.find_good_parse_start` and add tests for indent function.
```

Comment:
I don't think the `meth` role should be used here -- no link will be generated.

Reply:
I don't think we would want a link anyway.  I am revising the blurb.